boards/posix/simple_process/doc/board.rst

```diff
@@ -0,0 +1,179 @@
+
+.. _simple_process:
+
+Native POSIX execution (simple_process)
+#######################################
+
+Overview
+********
+
+Using this board, a Zephyr application can be compiled together with
+the Zephyr kernel, creating a normal console executable that runs as
+a native application on the host OS, without emulation.  Instead,
+you use native host tools for compiling, debugging, and analyzing your
+Zephyr application, eliminating the need for architecture-specific
+target hardware in the early phases of development.
+
+Host system dependencies
+========================
+
+This port is designed to run in POSIX compatible operating systems.
+It has only been tested on Linux, but should also be compatible with macOS.
+
+.. note::
+
+   Remember that you must have the 32-bit C library installed in your system
```

Comment:
Reading this again, saying "remember" implies you've told them this before.  So let's just say, ``You must have ...``